lib/tower_cli/resources/job.py

```diff
@@ -89,6 +90,8 @@ def launch(self, job_template=None, tags=None, monitor=False, timeout=None,
         # Get the job template from Ansible Tower.
         # This is used as the baseline for starting the job.
 
+        tags = kwargs.get('tags', None)
```

Comment:
@AlanCoding Great idea! But since this will be a pure restructure unrelated to any specific issue, shall we make an isolated PR for that after this one is merged?